client/post-editor/editor-revisions-list/style.scss

```diff
@@ -3,21 +3,16 @@
 @import 'components/accordion/style';
 
 .editor-revisions-list {
-	position: absolute;
-	top: 0;
-	right: 0;
+	position: relative;
 	background: $gray-light;
-
-	@include breakpoint( '<660px' ) {
-		left: 0;
-		height: 120px;
-	}
+	flex-basis: 120px;
```

Comment:
On mobile the flex-basis defines the height thanks to `flex-direction: column-reverse;`